target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -182,7 +182,8 @@ cf-e320n-v2)
 	ucidef_set_led_wlan "wlan" "WLAN" "$board:blue:wlan" "phy0tpt"
 	;;
 cf-e380ac-v1|\
-cf-e380ac-v2)
+cf-e380ac-v2)\
+cf-e355ac)
 	ucidef_set_led_netdev "lan" "LAN" "$board:green:lan" "eth0"
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "$board:blue:wlan2g" "phy1tpt"
```

Comment:
I will redefine the use of led defined in mach-*.c file. Thank you very much for your advice.